sections/main-search.liquid

```diff
@@ -98,7 +98,8 @@
                     media_size: section.settings.image_ratio,
                     show_secondary_image: section.settings.show_secondary_image,
                     enable_image_padding: section.settings.enable_image_padding,
-                    show_vendor: section.settings.show_vendor
+                    show_vendor: section.settings.show_vendor,
+                    show_image_outline: section.settings.show_image_outline
```

Comment:
Off-topic: can we fix the "add padding" setting? On the search template, it should call `add_image_padding` instead of `enable_image_padding`.

![image](https://user-images.githubusercontent.com/658169/126683249-9889cbfe-7e04-4fc2-b1e8-c3548a2c04a0.png)

Reply:
This was fixed in another PR, which has now been merged to `main`. https://github.com/Shopify/dawn/commit/664071cac6609205e1647524062d7e7c717ba0b4